coral/index.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700" />
+<head>
```

Comment:
I think the formatting got messed up a bit in the last version. We have no set formatter for this, but my editor is set up to use 2 spaces :)

Reply:
Oh, seems like my IDE was still set to the 4 spaces default in html files. Updated it!

Reply:
The format still seems a bit off - e.g. there is no indentation in the first `div`